src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -568,5 +580,17 @@ private void CheckSize(int countToAdd)
                 }
             }
         }
+
+        private struct EncodingPreamblePair
```

Comment:
> Is this code ever supposed to run on x86? As is now CoreCLR doesn't quite support x86.

CoreCLR is meant to support 32-bit, at least on Windows, and in fact we currently run most of our tests on Windows on 32-bit (though hopefully they'll soon run 64-bit by default).  Plus many of these libraries are shared with .NET Native.